core/finality-grandpa/src/import.rs

```diff
@@ -590,9 +590,8 @@ where
 				info!(target: "finality", "Imported justification for block #{} that triggers \
 					command {}, signaling voter.", number, command);
 
-				if let Err(e) = self.send_voter_commands.unbounded_send(command) {
-					return Err(ConsensusError::ClientImport(e.to_string()).into());
-				}
+				// send the command to the voter
+				let _ = self.send_voter_commands.unbounded_send(command);
```

Comment:
This can happen if the GRANDPA voter/observer is not running (e.g. running with `--no-grandpa`). Either way, if the receiver is not there because it crashed then it will take the node down if it is a voter and if it's an observer we can safely ignore this error.

Reply:
This comment should be included in the source file.